src/main/java/io/supertokens/webserver/api/session/SessionAPI.java

```diff
@@ -78,27 +77,26 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
         JsonObject userDataInDatabase = InputParser.parseJsonObjectOrThrowError(input, "userDataInDatabase", false);
         assert userDataInDatabase != null;
 
-        boolean useStaticSigningKey = version.equals("2.18") ?
-                Boolean.TRUE.equals(InputParser.parseBooleanOrThrowError(input, "useStaticSigningKey", true)) :
-                Config.getConfig(main).getAccessTokenSigningKeyDynamic();
+        Boolean inputUseStaticKey = InputParser.parseBooleanOrThrowError(input, "useStaticSigningKey", true);
```

Comment:
I prefer to set this to a default value (true or false) instead of keeping it as null. This way, the intention is clear.